src/filter_noise.py

```diff
@@ -0,0 +1,35 @@
+from core_data_modules.logging import Logger
+from src.lib.pipeline_configuration import PipelineConfiguration
+
+log = Logger(__name__)
+
+
+class FilterNoise(object):
+    @staticmethod
+    def filter_noise_radio_messages(messages):
+        """
+        Filters a list of messages which have been labelled as Noise_Other_Project.
+
+        :param messages: List of message objects to filter.
+        :type messages: list of TracedData
+        :param message_keys: Keys in each TracedData to search for a message.
+        :type message_keys: list of str
+        :return: Filtered list.
+        :rtype: list of TracedData
+        """
+        not_noise = []
+        log.info(len(messages))
+        for plan in PipelineConfiguration.RQA_CODING_PLANS:
+            for td in messages:
+                if plan.binary_coded_field is not None and td[plan.binary_coded_field]["CodeID"] != "code-NOP-4eb70633" and \
```

Comment:
This hard-codes the NOP id into the pipeline, so this function will silently stop working if we ever use a code scheme which uses a different id for NOP. Instead, it would be better to use something like `plan.binary_code_scheme.get_code_with_control_code(Codes.NOISE_OTHER_PROJECT)` (writing that from memory, so fn names may not be exactly accurate but should give the gist).